apps/web/src/views/PoolDetail/components/PoolTvlWarnning.tsx

```diff
@@ -0,0 +1,30 @@
+import { useTranslation } from '@pancakeswap/localization'
+import { Message, MessageText, Text } from '@pancakeswap/uikit'
+import { PoolInfo } from 'state/farmsV4/state/type'
+import { TextLink } from 'views/Ifos/components/IfoCardStyles'
+import { useRouterQuery } from '../hooks/useRouterQuery'
+
+const ONE_BILLION = 1_000_000_000
+export const PoolTvlWarning = ({ poolInfo }: { poolInfo: PoolInfo }) => {
+  const { t } = useTranslation()
+  const { id } = useRouterQuery()
+  const tvlUsd = parseFloat(poolInfo.tvlUsd ?? '0')
+  if (tvlUsd < ONE_BILLION) {
+    return null
+  }
+  // const pairAddress = Pair.getAddress(token0.wrapped, token1.wrapped)
+  const link = `https://pancakeswap.finance/info/v3/pairs/${id}`
```

Comment:
it could be v2/SS and could be different chain

Reply:
done